### DISCUSSION TITLE
why array use << but chan use <-

### DISCUSSION THREAD

**Discussion by Silentdoer** [vlang/v#15264] [Ideas]
it is not consistently?

**Comment by Wajinn** [comment]
Actually, the difference can visually prevent confusion where one is working with arrays and not channels.  If they were the same, probably would get more of the opposite complaint.  Think about if you never saw vlang before or were not familiar with arrays and channels, would likely create confusion looking through code and trying to differentiate.

Also, channels and concurrency comes from the Go heritage, so makes more sense to use a similar syntax when dealing with such and make it easier for those familiar with it to transition over to vlang.
